src/main/webapp/functions.js

```diff
@@ -37,8 +37,4 @@ function makeGraph() {
       );
       diffChart.draw(difficultyData, diffOptions);
     });
-}
```

Comment:
do you used the functions.js file anywhere almost certain I delete it one of my PRs

Reply:
i don't use it at all, not sure why it's even in here. I'll remove